components/home/Header/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Image from "next/image";
+import {
+  Container,
+  Logo,
+  SocialMediaLinks,
+  Navbar,
+  WatchMyStory,
+  NavWrapper,
+} from "./styled";
+
+interface HeaderProps {}
+
+const Header = ({}: HeaderProps) => {
+  return (
+    <Container>
+      <NavWrapper>
+        <Logo>
+          <h1>
```

Comment:
the H1 should be part of the logo component. Actually I'm thinking the lgoo should be a png